ext/dom/tests/gh16336_2.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+GH-16336 (Attribute intern document mismanagement)
+--EXTENSIONS--
+dom
+--FILE--
+<?php
+
+$doc = new DOMDocument();
+$elem = new DOMElement("g");
+$attr = new DOMAttr("iF", "j");
+
+$elem->setAttributeNode($attr);
```

Comment:
it took consulting an external tool (https://www.diffchecker.com/) to figure out that the difference between the _1 and _2 tests was the order of these two statements - suggest adding a comment to make it clearer to the next person that passes along